ts/BUILD.bazel

```diff
@@ -31,13 +75,58 @@ config_setting(
     flag_values = {":skipLibCheck": "honor_tsconfig"},
 )
 
+# TODO(2.0): remove this
+config_setting(
+    name = "legacy_verbose",
+    define_values = {"VERBOSE_LOGS": "1"},
+)
+
+config_setting(
+    name = "verbose_flag",
+    flag_values = {
+        ":verbose": "true"
+    }
+)
+
+config_setting(
+    name = "supports_workers_flag",
+    flag_values = {
+        ":supports_workers": "true"
```

Comment:
TODO: 2.0 change default to false? (assuming we are able to make swc default transpiler)

Reply:
i'll put up a comment above